tests/utils.go

```diff
@@ -769,6 +771,65 @@ func BeforeTestSuitSetup() {
 	SetDefaultEventuallyPollingInterval(defaultEventuallyPollingInterval)
 
 	AdjustKubeVirtResource()
+
+	// Make sure kubectl cache is updated with vm/vmi resources
```

Comment:
@rmohr what do you think about this piece of code to make sure the cache includes vm and vmi resources (or maybe other resources that might be affected by this issue) instead of messing with permissions?
I noticed there were several kinds of cache, and only one of them is controlled with `--cache-dir`.

Reply:
Sorry for the late reply. Yes that may be a simple alternative. We could actually run it before each test. I think that you don't want to fail on errors if they fail. You may also want to limit it to only one retry, since it should work on the second try. We don't want to hide similar looking but different failures.